skimage/util/_invert.py

```diff
@@ -2,15 +2,25 @@
 from .dtype import dtype_limits
 
 
-def invert(image):
+def invert(image, signed_float=False):
     """Invert an image.
 
-    Substract the image to the maximum value allowed by the dtype maximum.
+    Image inversion. If the image is of type unsignedinteger, the image
```

Comment:
I'd make this as short and informative as possible:
- remove `Image inversion. `. It is already mentioned above;
- use ` ``image`` ` instead of `the image`;
- true -> True, false -> False;
- make type description consistent: `type signedinteger`, `float type` -> `signedinteger type`, `float type` or v.v..

Moreover, I think, this might look better in the description of the returned value (below `invert : ndarray`).